<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisComp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Usar_Resultado">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Usar Resultado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>541</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagem 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagem 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_limiar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>480</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIn_cio">
    <property name="title">
     <string>Início</string>
    </property>
    <addaction name="actionAbrir_Imagem_1"/>
    <addaction name="actionAbrir_Imagem_2"/>
    <addaction name="actionSalvar_Resultado"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuIn_cio"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir_Imagem_1">
   <property name="text">
    <string>Abrir Imagem 1</string>
   </property>
  </action>
  <action name="actionAbrir_Imagem_2">
   <property name="text">
    <string>Abrir Imagem 2</string>
   </property>
  </action>
  <action name="actionSalvar_Resultado">
   <property name="text">
    <string>Salvar Resultado</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionTons_de_Cinza">
   <property name="text">
    <string>Tons de Cinza</string>
   </property>
  </action>
  <action name="actionBin_rio">
   <property name="text">
    <string>Binário</string>
   </property>
  </action>
  <action name="actionSoma">
   <property name="text">
    <string>Soma</string>
   </property>
  </action>
  <action name="actionSubtra_o">
   <property name="text">
    <string>Subtração</string>
   </property>
  </action>
  <action name="actionMult">
   <property name="text">
    <string>Multiplicação</string>
   </property>
  </action>
  <action name="actionDivisao">
   <property name="text">
    <string>Divisão</string>
   </property>
  </action>
  <action name="actionAnd">
   <property name="text">
    <string>And</string>
   </property>
  </action>
  <action name="actionOr">
   <property name="text">
    <string>Or</string>
   </property>
  </action>
  <action name="actionNot">
   <property name="text">
    <string>Not</string>
   </property>
  </action>
  <action name="actionXor">
   <property name="text">
    <string>Xor</string>
   </property>
  </action>
  <action name="actionEscalonamento">
   <property name="text">
    <string>Escalonamento</string>
   </property>
  </action>
  <action name="actionTransla_o">
   <property name="text">
    <string>Translação</string>
   </property>
  </action>
  <action name="actionRota_o">
   <property name="text">
    <string>Rotação</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionKirsch">
   <property name="text">
    <string>Kirsch</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionPrimeira_Ordem">
   <property name="text">
    <string>Primeira Ordem</string>
   </property>
  </action>
  <action name="actionSegunda_Ordem">
   <property name="text">
    <string>Segunda Ordem</string>
   </property>
  </action>
  <action name="actionPrimeira_Ordem_2">
   <property name="text">
    <string>Primeira Ordem</string>
   </property>
  </action>
  <action name="actionSegunda_Ordem_2">
   <property name="text">
    <string>Segunda Ordem</string>
   </property>
  </action>
  <action name="actionOp_es">
   <property name="text">
    <string>Opções</string>
   </property>
  </action>
  <action name="actionGaussiano">
   <property name="text">
    <string>Gaussiano</string>
   </property>
  </action>
  <action name="action3x3">
   <property name="text">
    <string>3x3</string>
   </property>
  </action>
  <action name="action5x5">
   <property name="text">
    <string>5x5</string>
   </property>
  </action>
  <action name="action9x9">
   <property name="text">
    <string>9x9</string>
   </property>
  </action>
  <action name="actionHistogramas">
   <property name="text">
    <string>Histograma</string>
   </property>
  </action>
  <action name="actionAuto_Escala">
   <property name="text">
    <string>Auto Escala</string>
   </property>
  </action>
  <action name="actionEqualiza_o">
   <property name="text">
    <string>Equalização</string>
   </property>
  </action>
  <action name="actionLimiariza_oGlobal">
   <property name="text">
    <string>Limiarização Global</string>
   </property>
  </action>
  <action name="actionLimiariza_o_Otsu">
   <property name="text">
    <string>Limiarização Otsu</string>
   </property>
  </action>
  <action name="actionM_dia">
   <property name="text">
    <string>Média</string>
   </property>
  </action>
  <action name="actionMediana">
   <property name="text">
    <string>Mediana</string>
   </property>
  </action>
  <action name="actionLaplaciano">
   <property name="text">
    <string>Laplaciano</string>
   </property>
  </action>
  <action name="actionPassa_Alta">
   <property name="text">
    <string>Passa Alta</string>
   </property>
  </action>
  <action name="actionDilata_o">
   <property name="text">
    <string>Dilatação</string>
   </property>
  </action>
  <action name="actionEros_o">
   <property name="text">
    <string>Erosão</string>
   </property>
  </action>
  <action name="actionAbertura">
   <property name="text">
    <string>Abertura</string>
   </property>
  </action>
  <action name="actionFechamento">
   <property name="text">
    <string>Fechamento</string>
   </property>
  </action>
  <action name="actionOrienta_o">
   <property name="text">
    <string>Orientação</string>
   </property>
  </action>
  <action name="actionDimens_es">
   <property name="text">
    <string>Dimensões</string>
   </property>
  </action>
  <action name="actionPosi_ao">
   <property name="text">
    <string>Posição</string>
   </property>
  </action>
  <action name="action_rea">
   <property name="text">
    <string>Área</string>
   </property>
  </action>
  <action name="actionSegmenta_o">
   <property name="text">
    <string>Segmentação</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
